Main/ClientApp/src/Infrastructure/FetchUtilities.ts

```diff
@@ -8,6 +8,11 @@ export async function authorizedGet(apiEndpoint: string) {
   });
 }
 
+export async function unauthorizedGet(apiEndpoint: string) {
```

Comment:
I don't like this name. Unauthorized means the request was not authorized by the server.
Besides, whats the point of this method? why not just use await fetch() in the code? the authorized... were nice because they contained more logic, here it is just unnecessary code

Reply:
true